src/lib/experimental/fetch/icp.ts

```diff
@@ -61,7 +61,8 @@ export async function fetchIcp(
         }
     );
 
-    const decodedResult = IDL.decode(funcIdl.retTypes, result);
+    const resultBytes: Uint8Array<ArrayBuffer> = new Uint8Array(result);
+    const decodedResult = IDL.decode(funcIdl.retTypes, resultBytes.buffer);
```

Comment:
Make this like the others, : ArrayBuffer and use .buffer on the right hand side

Reply:
Done